.gitignore

```diff
@@ -1,4 +1,7 @@
 #Files to be ignored in the commits
 
 # dependencies
-/node_modules
\ No newline at end of file
+/node_modules
+
```

Comment:
- [ ] Kindly remove the empty line for a clean code and remove the MIT.md as well. You are required to ignore the test.md and node_modules.

Reply:
Thank you. it's done.